### PR TITLE
small modification

### PR DESCRIPTION
**Pull request by gk-sk**

### CODE REVIEW

**Review by Gk1sK — APPROVED** [review]

**Review by Gk1sK — CHANGES_REQUESTED** [review]
add some more text

File: web-pages/new-feature.html [hunk 1]
```diff
@@ -119,7 +119,7 @@
 	<div class="inner-content col-md-12">
 	<div class="container">
 
-	   <h1>WATCH THIS VIDEO</h1>
+	   <h1>WATCH THIS VIDEO. coz, this is is really funny and hilarious.</h1>
```
Comment (Gk1sK) [inline web-pages/new-feature.html:122]: add some more text.

**Review by Gk1sK — CHANGES_REQUESTED** [review]

File: web-pages/new-feature.html [hunk 2]
```diff
@@ -119,7 +119,7 @@
 	<div class="inner-content col-md-12">
 	<div class="container">
 
-	   <h1>WATCH THIS VIDEO</h1>
+	   <h1>WATCH THIS VIDEO. coz, this is is really funny and hilarious and little longer.</h1>
```
Comment (Gk1sK) [inline web-pages/new-feature.html:122]: reduce the text.

**Review by Gk1sK — CHANGES_REQUESTED** [review]

File: web-pages/new-feature.html (hunk 2 above)
Comment (gk-sk) [reply web-pages/new-feature.html]: reduced as requested.

**Review by Gk1sK — APPROVED** [review]

**Pull request merged**